app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -1379,9 +1379,15 @@ class BrowserTabViewModelTest {
         assertNull(testee.ctaViewState.value!!.cta)
     }
 
+    @Test
+    fun whenSurveyCtaDismissedAndCovidCtaNotShownthenCtaIsCovid() {
```

Comment:
I would add `andWidgetNotCompatible` or similar, because we are testing that the survey has been dismissed, but no info about the widget CTA.

Reply:
Done!